www/www-deploy.yml

```diff
@@ -141,16 +135,58 @@
       dest: /careers
     - match: "~/site/community/scripts/?$"
       dest: https://docs.openmicroscopy.org/latest/omero/developers/scripts/index.html
-    - match: "~/site/community/(?<link>.*)$"
-      dest: https://www-legacy.openmicroscopy.org/site/community/$link
 
     # support
     - match: "~/site/support/?$"
       dest: /docs
     - match: "~/site/support/ome-artwork(/.*)?$"
       dest: /artwork
-    - match: "~/site/support/(?<link>.*)$"
-      dest: https://www-legacy.openmicroscopy.org/site/support/$link
+
+    # info
+    - match: "~/info/vulnerabilities/?$"
+      dest: /security/advisories/
+    - match: "~/info/vulnerabilities/(?<link>.*[^/])/?$"
+      dest: /security/advisories/$link/
+    - match: "~/info/omero$"
```

Comment:
The trailing `omero$` fails the test when `/info/omero/` is passed as the URL suffix. Since we are matching regexp, could be turned into `~/info/omero/?$`

Reply:
Also should we handle the capitalized version as well (to match the info redirects below)?

Reply:
OMERO capitalized but without a client or server added to it isn't on the list of info URLs we've ver given out - I didn't want to use this PR to add new redirects